src/models/transactions/common.ts

```diff
@@ -79,7 +79,7 @@ export interface BaseTransaction {
   Flags?: number | GlobalFlags;
   LastLedgerSequence?: number;
   Memos?: Array<{ Memo: Memo }>;
-  Signers?: Signer[];
+  Signers?: Array<{ Signer: Signer }>;
```

Comment:
Instead of this, I would change the signer object to be `Signer: {...whatever-was-there-before}`